apps/fluent-tester/src/TestComponents/Menu/MenuScrollView.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { ButtonV1 as Button } from '@fluentui/react-native';
+import { Menu, MenuItem, MenuTrigger, MenuPopover, MenuList } from '@fluentui-react-native/menu';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { TextInput, StyleSheet, View } from 'react-native';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+
+const styles = StyleSheet.create({
+  menuContainerRow: { flexDirection: 'row' },
+  menuMargin: { marginRight: 20 },
+});
+
+export const MenuScrollView: React.FunctionComponent = () => {
+  const [maxHeight, setMaxHeight] = useState<number>(800);
+  const ScrollViewMenuPopover = MenuPopover.customize({ maxHeight: maxHeight });
+  const menuItems = [];
+  const [data, setData] = useState([
+    <MenuItem key={1}>MenuItem 1</MenuItem>,
+    <MenuItem key={2}>MenuItem 2</MenuItem>,
+    <MenuItem key={3}>MenuItem 3</MenuItem>,
+    <MenuItem key={4}>MenuItem 4</MenuItem>,
+    <MenuItem key={5}>MenuItem 5</MenuItem>,
+  ]);
+
+  for (let i = 0; i < 400; i++) {
+    menuItems.push(<MenuItem key={i}>MenuItem {i}</MenuItem>);
+  }
```

Comment:
Nit: My first read through, I thought we were pushing 400 objects to `data` not `menuItem` because of the ordering of the variable declarations. I think I would prefer to keep all hook calls at the top of the function, and to have `const menuItems = []` defined right before the for loop.